src/Leaderboard.js

```diff
@@ -0,0 +1,23 @@
+export default class Leaderboard {
+  #BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+  #GAME_ID = '6RWYo73TvPoVLbLnw9Tv';
+
+  async getResults() {
+    const url = `${this.#BASE_URL}games/${this.#GAME_ID}/scores/`;
+    const response = await fetch(url);
+    return response.json();
+  }
+
+  async postResult(data) {
```

Comment:
- [ ] As mentioned above Let's use `arrow functions` instead of the standard functions.